### DISCUSSION TITLE
error make compiled

### DISCUSSION THREAD

**Discussion by medioman** [aperezdc/termite#23] [Q&A] [answered]
hi 
have this error:
termite-16.9/subprojects/vte/src/vte/meson.build:61:15: ERROR: Program 'glib-mkenums' not found or not executable
archlinux

**Comment by hurufu** [comment, marked as answer]
You can try to install `glib2-devel` package:

```shell
sudo pacman -S glib2-devel
```

**Reply by aperezdc** [reply]
Indeed, I just [fixed the AUR PKGBUILD](https://aur.archlinux.org/cgit/aur.git/commit/PKGBUILD?h=termite&id=791bed3bcc4e000bc9874c035bbfdb015f9e5e2c) to include `glib2-devel`. The Arch package for GLib has split the development tools into `glib2-devel` a while ago.
